firebase-components/src/main/java/com/google/firebase/components/compat/ContainerOwner.java

```diff
@@ -0,0 +1,183 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.components.compat;
+
+import android.annotation.TargetApi;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Build;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.UserManager;
+import androidx.annotation.GuardedBy;
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import androidx.collection.ArrayMap;
+import com.google.firebase.components.Component;
+import com.google.firebase.components.ComponentDiscovery;
+import com.google.firebase.components.ComponentRuntime;
+import com.google.firebase.components.ContainerInfo;
+import com.google.firebase.components.Preconditions;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Executor;
+import java.util.concurrent.atomic.AtomicReference;
+
+/** */
+public final class ContainerOwner {
+
+  private static final @NonNull String DEFAULT_CONTAINER_NAME = "[DEFAULT]";
```

Comment:
This value is duplicated in FirebaseApp. Make it public here and refer to it in FirebaseApp?